tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/SeqScanExec.java

```diff
@@ -69,9 +70,8 @@ public SeqScanExec(TaskAttemptContext context, ScanNode plan,
     this.qual = plan.getQual();
     this.fragments = fragments;
 
-    if (fragments != null
-        && plan.getTableDesc().hasPartition()
```

Comment:
Even fragments is null, partition columns in expr should be replaced with null. Without this, some tests fails with exception("No Such Column Reference").